lib/ical/recur_iterator.js

```diff
@@ -843,11 +843,11 @@ ICAL.RecurIterator = (function() {
         this.last.month = next.month;
     },
 
-    ruleDayOfWeek: function ruleDayOfWeek(dow) {
+    ruleDayOfWeek: function ruleDayOfWeek(dow, aFirstDayOfWeek) {
```

Comment:
`aWeekStart` should be used everywhere to be consistent. Maybe you want to add documentation to this function.